content/guide/02-template-syntax.md

```diff
@@ -263,7 +263,7 @@ If the expression in `{#await expression}` *isn't* a promise, Svelte skips ahead
 
 ### Directives
 
-The last place where Svelte template syntax differs from regular HTML: *directives* allow you to add special instructions for adding [event handlers](guide#event-handlers), [two-way bindings](guide#two-way-binding), [refs](guide#refs) and so on. We'll cover each of those in later stages of this guide – for now, all you need to know is that directives can be identified by the `:` character:
+The last place where Svelte template syntax differs from regular HTML: *directives* allow you to add special instructions for adding [event handlers](guide#event-handlers), [binding data](guide#binding-data), [refs](guide#refs) and so on. We'll cover each of those in later stages of this guide – for now, all you need to know is that directives can be identified by the `:` character:
```

Comment:
While you're at it, could we change `[refs]` to `[referencing elements]`, the action verb which both matches the previous two and also makes sense to someone new to Svelte?

Reply:
I've made that edit